plugins/woocommerce/client/legacy/js/admin/meta-boxes-product-variation.js

```diff
@@ -1284,7 +1285,16 @@ jQuery( function ( $ ) {
 					if ( value != null ) {
 						data.value = value;
 					} else {
-						return;
+						cancel = true;
+					}
+					break;
+				case 'variable_unset_cogs_value':
+					if(
```

Comment:
```suggestion
					if (
```

Reply:
These and the other linting issues mentioned are fixed in [`432499d` (#54399)](https://github.com/woocommerce/woocommerce/pull/54399/commits/432499d06c88b66b909bfa8b2ac9c511e42b2011).